dask_jobqueue/sge.py

```diff
@@ -40,32 +40,31 @@ def __init__(
 
         header_lines = []
         if self.job_name is not None:
-            header_lines.append("#$ -N %(job-name)s")
+            header_lines.append("#$ -N %s" % self.job_name)
```

Comment:
I agree this would be better, but we should probably do this for every other schedulers as you noticed. So I'll keep that for another PR if we decide to do it.